LoopBackTests/server/index.js

```diff
@@ -87,10 +83,11 @@ Widget.testGeoPoint = function(geopoint, callback) {
   geopoint.lng += 1;
   callback(null, geopoint);
 };
-Widget.testGeoPoint.shared = true;
-Widget.testGeoPoint.accepts = [{ arg: 'geopoint', type: 'geopoint' }];
-Widget.testGeoPoint.returns = [{ arg: 'geopoint', type: 'geopoint' }];
-Widget.testGeoPoint.http = { verb: 'get' };
+Widget.remoteMethod('testGeoPoint', {
+  accepts: [{ arg: 'geopoint', type: 'geopoint' }],
+  returns: [{ arg: 'geopoint', type: 'geopoint' }],
+  http: { verb: 'get', source: 'form'}
```

Comment:
AFAIK, the `GET` method explicitly forbids non-empty request bodies, therefore there will be no `form` to get the value from.

I assume the tests are passing for you, which may mean that `source: 'form'` is ignored, or that the test is sending a `GET` request with a non-empty body. Could you please investigate?